src/config/dark-sites.config

```diff
@@ -1,3 +1,5 @@
+!blog.discordapp.com
```

Comment:
What's this?

Reply:
That has to be pushed to the URL Patch woops